db/seeds/run-seed-test.js

```diff
@@ -0,0 +1,9 @@
+const testData = require('../data/test-data/index.js');
+const seed = require('./seed.js');
+const db = require('../connection.js');
+
+const runSeed = () => {
+  return seed(testData).then(() => db.end());
```

Comment:
I think you've confused beforeEach, and afterAll here! We need to tell our code that after it has run all of our tests, it should end the connection to the database. Separately, we need to say that beforeEach time we run our tests, we want to re-seed the database, to ensure our data isn't persisted when we do things like add new pieces of information that might get in the way of some of our other assertions.